client/my-sites/stats/stats-subscribers-page/subsriber-highlights.jsx

```diff
@@ -0,0 +1,97 @@
+import { HighlightCard, Popover } from '@automattic/components';
+import { Icon, info } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import '../wordads/highlights-section.scss';
```

Comment:
Could we use a `subscriber-section.scss` instead of using styling from `wordads`? This import creates an unnecessary dependency chain between subscriber and wordads sections.

Reply:
Great catch. If we need to share the same styles, we can create a mixin and import it the same way we did for all of the grids in `grid-layout` or `modernized-mixins`. Mixins can also accept parameters and are very flexible.

Reply:
I've copied the necessary styles into a new CSS file for now. I want to try a small refactoring next which should eliminate the need for a mixin.